primitives/account/src/account/staking_contract/validator.rs

```diff
@@ -629,9 +653,9 @@ impl StakingContract {
         // Check that the validator is not currently jailed.
         if validator.is_jailed(block_number) {
             debug!(
-                "Validator {} is jailed until {}",
+                "Validator {} is jailed since {}",
```

Comment:
use `tracing` style:
```suggestion
               ?validator.address,
               jailed_from = validator.jailed_from.unwrap(),
                "Validator is jailed",
```